core/src/main/java/nu/yona/server/subscriptions/service/BuddyDisconnectMessageDTO.java

```diff
@@ -61,10 +61,9 @@ public boolean isProcessed()
 	public static BuddyDisconnectMessageDTO createInstance(UserDTO actingUser, BuddyDisconnectMessage messageEntity)
 	{
 		User userEntity = messageEntity.getUser(); // may be null if deleted
-		UserDTO user = userEntity != null ? UserDTO.createInstance(userEntity) : UserDTO.createRemovedUserInstance();
```

Comment:
If you just retain the below line, the API contract for ``createInstance`` can be regular:
```
UserDTO user = userEntity == null ? null : UserDTO.createInstance(userEntity);
```